uiowa/uiowa
# P2lb Conversion Status column

I wanted a way to know if a page had issues with conversion really quickly instead of having to go to every node to check, so I added a column to the converter page that checks for issues.

![Screenshot 2023-08-21 at 11 27 54 AM](https://github.com/uiowa/uiowa/assets/23122142/af0b988c-2e17-4b1d-81af-8be539d4952b)


# How to test

1. Sync a V2 site
2. Enable P2lb
3. Check the converter page
4. Investigate the links and whatnot and if it is evaluating correctly.
